### PR TITLE
Hpearson/use enum for combobox

### PR DESCRIPTION
**Pull request by harrytpstudent**
Added enumeration type for the different module types.

### CODE REVIEW

**Review by MalawiGongo — APPROVED** [review]
Looks good to me

**Comment by harrytpstudent** [comment]
I've added a new class to separate out the responsibility of creating modules.

File: PatientMonitoringSystem/PatientMonitoringSystem/Enums/ModuleTypeEnum.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PatientMonitoringSystem.Enums
+{
+	public enum ModuleType
```
Comment (spblackford) [inline PatientMonitoringSystem/PatientMonitoringSystem/Enums/ModuleTypeEnum.cs]: @harrytpstudent No need to call the class "ModuleTypeEnum". Just "ModuleType" would do.

Comment (harrytpstudent) [reply PatientMonitoringSystem/PatientMonitoringSystem/Enums/ModuleTypeEnum.cs]: Sure.  Will amend now.

File: PatientMonitoringSystem/PatientMonitoringSystem/PatientMonitoringSystem.csproj [hunk 2]
```diff
@@ -47,6 +47,8 @@
     <Reference Include="System.Xml" />
   </ItemGroup>
   <ItemGroup>
+    <Compile Include="Enums\ModuleType.cs" />
```
Comment (spblackford) [inline PatientMonitoringSystem/PatientMonitoringSystem/PatientMonitoringSystem.csproj:50]: Good to see this class renamed. :-)

File: PatientMonitoringSystem/PatientMonitoringSystem/Views/BedsideSystemView.cs [hunk 3]
```diff
@@ -30,14 +31,8 @@ private void Updater_Tick(object sender, EventArgs e)
...
-			if (ModuleCombo.SelectedIndex == -1)
-			{
-				MessageBox.Show("Please select a module type!");
-			}
-			else {
-				var strategyType = ModuleCombo.SelectedItem.ToString();
-				controller.AddModule(moduleName, strategyType);
-			}
+			ModuleType moduleType = (ModuleType)ModuleCombo.SelectedItem;
+			controller.AddModule(moduleName, moduleType);
```
Comment (spblackford) [inline PatientMonitoringSystem/PatientMonitoringSystem/Views/BedsideSystemView.cs:35]: @harrytpstudent Will this blow up if the user has not selected any option from the dropdown menu?

Comment (spblackford) [reply PatientMonitoringSystem/PatientMonitoringSystem/Views/BedsideSystemView.cs]: Test

Comment (spblackford) [reply PatientMonitoringSystem/PatientMonitoringSystem/Views/BedsideSystemView.cs:35]: Looks like the control is safe from bad user input.

**Review by spblackford — APPROVED** [review]

**Pull request merged**